### DISCUSSION TITLE
MobileSafeArea - Lobe UI

### DISCUSSION THREAD

**Discussion by giscus** [lobehub/lobe-ui#172] [Q&A]
# MobileSafeArea - Lobe UI

https://ui.lobehub.com/components/mobile-safe-area

**Comment by adntin** [comment]
DependencyNotFoundError
Could not find dependency: 'antd-style' relative to '/node_modules/@lobehub/ui/es/Icon/style.js'
Suggested solutions:
Add antd-style as dependency
▶ 23 stack frames were collapsed.
$csb$eval
/App.tsx:1
> 1 | import { MobileSafeArea } from '@lobehub/ui';
  2 | import { Flexbox } from 'react-layout-kit';
  3 | 
  4 | export default () => {
View compiled
▶ 4 stack frames were collapsed.
$csb$eval
/index.tsx:10
   7 | 
   8 | import React from 'react';
   9 | import { createRoot } from "react-dom/client";
> 10 | import App from "./App";
  11 | 
  12 | const rootElement = document.getElementById("root");
  13 | const root = createRoot(rootElement);
View compiled
▶ 7 stack frames were collapsed.
This screen is visible only in development. It will not appear if the app crashes in production.
Open your browser’s developer console to further inspect this error.
This error overlay is powered by `react-error-overlay` used in `create-react-app`.
